plexapi/playqueue.py

```diff
@@ -175,8 +175,11 @@ def create(
             args["uri"] = "library:///directory/{uri_args}".format(uri_args=uri_args)
             args["type"] = items[0].listType
         elif items.type == "playlist":
-            args["playlistID"] = items.ratingKey
             args["type"] = items.playlistType
+            if items.radio:
+                args["uri"] = f"server://{server.machineIdentifier}/{server.library.identifier}{items.key}"
```

Comment:
Use `%s` string formatting to be consistent with the rest of Plex API?
```suggestion
                args["uri"] = "server://%s/%s%s" % (
                    server.machineIdentifier, server.library.identifier, items.key
                )
```

Reply:
I missed all the other strings in `playqueue` all use `.format()`.

Reply:
I'd prefer to use f-strings moving forward. Most string formatting in the project was written before we could use the feature in all supported Python versions. Now that 3.5 support has dropped they can be used safely.

Reply:
I would too, but I want to do it all in one go.

Reply:
It's a style change that doesn't affect functionality. Let's require f-strings for any additions/changes moving forward and convert existing formatting in followup PRs when time/effort permits.